src/app.ts

```diff
@@ -1,121 +1,35 @@
-import express, { Request, Response, NextFunction } from 'express';
-
-// // Joi validation schema's
-import { 
-  validateGetConfigRequest, 
-  validatePutConfigRequest, 
-  validateAddWorkdaysRequest, 
-  validateIsWorkdayRequest 
-} from './validation/validators.js';
+import express from 'express';
 
 // business logic classes
-import Workdays from './Workdays.js';
-import DiskCache from './DiskCache.js';
-import Config from './Config.js';
+import Workdays from './Workdays';
+import DiskCache from './DiskCache';
+import Config from './Config';
 
-import { holidays as nl } from './lib/zones/nl.js';
-import { holidays as be } from './lib/zones/be.js';
-import ZoneNotFoundError from './errors/ZoneNotFoundError.js';
+import registerRoutes from './middleware/registerRoutes';
+import initializeSwagger from './middleware/initializeSwagger';
+import handleFailures from './middleware/handleFailures';
 
 // create the express application
 const app = express();
 app.use(express.json());
 
+initializeSwagger(app);
+
 // Setup the application
 const cache = new DiskCache();
```

Comment:
Mogelijkheid om disk cache pad te overschrijven.
```suggestion
const cache = new DiskCache(DISK_CACHE_PATH);
```